R/utils.R

```diff
@@ -0,0 +1,49 @@
+#' Create a R object from a JSON string. Can be used similar to `sprintf`.
+#'
+#' @param x The JSON string (with placeholders)
+#' @param ... The values to be inserted into the placeholders
+#' @return The R object
+from_json_string <- function(x, ...) {
```

Comment:
given that this is now unused to my understanding, we can theoretically simply removing it again as it serves no flowr-specific purpose.

Reply:
Resolved in 960c7f6